zebra-network/src/meta_addr/tests/vectors.rs

```diff
@@ -34,3 +45,129 @@ fn sanitize_extremes() {
         check::sanitize_avoids_leaks(&max_time_entry, &max_sanitized);
     }
 }
+
+/// Test if new peer is not recently reachable.
+#[test]
+fn new_peer_is_not_recently_reachable() {
+    zebra_test::init();
+
+    let address = SocketAddr::from(([192, 168, 180, 9], 10_000));
+    let peer = MetaAddr::new_alternate(&address, &PeerServices::NODE_NETWORK)
+        .into_new_meta_addr()
+        .expect("MetaAddrChange can't create a new MetaAddr");
+
+    assert!(!peer.was_recently_reachable());
```

Comment:
Possible Bug: Why is a new alternate peer considered unreachable for gossiping?

That seems a bit unexpected - can you fix the bug or add a comment to the test?

Reply:
Done in [749e66f](https://github.com/jvff/zebra/commit/749e66f55ea16c27fbbb6883a98a46bd84616f1a) and [a3654d2](https://github.com/jvff/zebra/commit/a3654d27b10d597869415276b2ab14a2d131aa63).